File: c++/exemplos/lacos/1060.cpp
#include <iostream>
#include <vector>
using namespace std;

int main() {
  double num;
  int i, sum;
  sum = 0;
  for (i = 1; i <= 6; i++)
  {
    cin >> num;
    if (num > 0)
    {
        sum = sum + 1;
    } 
  }
  cout << sum << " valores positivos" << endl;
  return 0;
} 

File: javascript/src/1060.js
export const getPositive = (numArr) => {
    const positive = [];
    for (let i = 0; i <= 6; i++) {
        if (numArr[i] > 0) {
            positive.push(numArr[i])
        }
    }
    return `${positive.length} valores positivos`
}